package/base-files/files/etc/uci-defaults/12_network-generate-ula

```diff
@@ -1,11 +1,9 @@
 [ "$(uci -q get network.globals.ula_prefix)" != "auto" ] && exit 0
 
-r1=$(dd if=/dev/urandom bs=1 count=1 |hexdump -e '1/1 "%02x"')
-r2=$(dd if=/dev/urandom bs=2 count=1 |hexdump -e '2/1 "%02x"')
-r3=$(dd if=/dev/urandom bs=2 count=1 |hexdump -e '2/1 "%02x"')
+r1=$(dd if=/dev/urandom bs=5 count=1 2>/dev/null |hexdump -ve '5/1 "%02x"')
```

Comment:
This could be improved further, please do: `hexdump -vn 5 -e '5/1 "%02x"' /dev/urandom`.

But overall, I agree with the spirit of this change!

Reply:
Aren't we super early in the boot and /dev/urandom is likely not super random ?

Reply:
I'm not sure how important that is seeing this for IPv6 ULA, it's probably random enough. If it ever became a problem maybe something like `dd if=/dev/urandom bs=1000 count=1 2>/dev/null | sha256sum` would work fine.

Reply:
A partial consideration for this change was that entropy might be starved at boot on some platforms, but I figured running this script means we're already past boot.

Reply:
> This could be improved further, please do: `hexdump -vn 5 -e '5/1 "%02x"' /dev/urandom`.
> 

That's quite good - though not everyone is familiar with hexdump, whereas dd is more ubiquitous, and easy to understand what's being piped to hexdump, at least.

Reply:
Amended.

Reply:
Welp, if you're not merging this, you can at least approve it. 🙏